requirements.pip

```diff
@@ -5,5 +5,5 @@ protobuf
 pytest
 six
 sympy
-tensorflow~=1.0
+tensorflow >= 1.5.0, <=1.15.0
```

Comment:
`1.15.3` seems to be the last TF1 release (https://pypi.org/project/tensorflow/1.15.3/)

Reply:
Done.